youtube_dl/extractor/polskieradio.py

```diff
@@ -15,6 +16,63 @@
 )
 
 
+class PolskieRadioProgrammeIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?polskieradio\.pl/(?P<id>\d+/\d+)(,.*)?(\?.*)?$'
```

Comment:
Sorry, in what way "I delegated extraction from one extractor to another" before this change?

Anyway, let me know if there are any other problems or if this is good to go now. Thank you for the review!